api/kyverno/v1/common_types.go

```diff
@@ -55,6 +55,11 @@ type WebhookConfiguration struct {
 	// MatchCondition configures admission webhook matchConditions.
 	// +optional
 	MatchConditions []admissionregistrationv1.MatchCondition `json:"matchConditions,omitempty" yaml:"matchConditions,omitempty"`
+
+	// WebhookTimeoutSeconds specifies the maximum time in seconds allowed to apply this policy.
+	// After the configured time expires, the admission request may fail, or may simply ignore the policy results,
+	// based on the failure policy. The default timeout is 10s, the value must be between 1 and 30 seconds.
+	WebhookTimeoutSeconds *int32 `json:"webhookTimeoutSeconds,omitempty" yaml:"webhookTimeoutSeconds,omitempty"`
```

Comment:
Please rename to `TimeoutSeconds` and the JSON to `timeoutSeconds`,  no need to repeat `webhook` as its in the parent name.

Reply:
do I need to change the name in crds too ?

Reply:
You will need to change the Golang structs and the run the make commands to generate the schema.

Reply:
should the names in the comments also be changes ? As I feel that for descriptiveness it should remain WebhookTimeOutSeconds only

Reply:
Yes. Please check other nested field in Kyverno or standard K8s resources as a guide.